sphinx_charts/charts.py

```diff
@@ -239,7 +239,7 @@ def setup(app):
             if "add_script_file" in dir(app):
                 app.add_script_file(path)
             else:
-                app.add_javascript(path)
+                app.add_js_file(path)
```

Comment:
To maintain compatibility with Sphinx 2, while preferably using the latest sphinx API on the first attempt, you'd do:
```py
try:
    app.add_js_file(path)
except AttributeError:
    # For backward compatibility with Sphinx 2 and 3
    app.add_javascript(path)
```